blockchain/types/accountkey/account_key.go

```diff
@@ -87,6 +87,8 @@ type AccountKey interface {
 	// IsCompositeType returns true if the account type is a composite type.
 	// Composite types are AccountKeyRoleBased and AccountKeyRoleBasedRLPBytes.
 	IsCompositeType() bool
+
+	IsContainedKey(recoveredKey *ecdsa.PublicKey) bool
```

Comment:
Let's use a more generalized method description.
```suggestion
    // ContainsPubkey returns true if the account type contains the given public key
    // in any part of its AccountKey.
	ContainsPubkey(pub *ecdsa.PublicKey) bool
```

Reply:
How do you think `IsContainedPubkey`? I also want to remove 'Is' prefix but we already have the `IsCompositeType` function returning boolean.